README.md

```diff
@@ -14,97 +14,85 @@ Compared to LuaJIT, Pallene aims to offer more predictable run-time performance.
-You can either install it [from source](https://www.lua.org/ftp/) or via the package manager for your Linux distro.
-If you install via the package manager then make sure to also install the Lua headers, which are often in a separate "development" package.
-
-After Lua 5.3 is installed, download the source code of LuaRocks from
-[https://github.com/luarocks/luarocks/releases](https://github.com/luarocks/luarocks/releases). Follow
-the build instructions appropriate for your platform and install it on your system.
+To check if you have installed the right version of Lua, run `lua -v`.
+It needs to say `Lua 5.x.x with core API`.
+If the message doesn't have the "with core API", that means you're using vanilla Lua.
 
-If LuaRocks is configured to use older versions of Lua, you may not be able to
-install the dependencies as described in the next section. Therefore, please
-configure LuaRocks to use Lua 5.3. You can use the following command to configure
-LuaRocks to use Lua 5.3 when compiling it:
-`./configure --lua-version=5.3`
+### Install Luarocks from source
 
-### Installing Lua dependencies
+You will probably also need to install Luarocks from source.
+We can't use the Luarocks from the package manager because that way it won't use the custom version of Lua we just installed.
 
-The easiest way to install these dependencies is with the [LuaRocks](http://luarocks.org) package manager:
+For more information on how to install Luarocks, please see the [Luarocks wiki](https://github.com/luarocks/luarocks/wiki/Installation-instructions-for-Unix).
+In the configure step, use the `--with-lua` flag to point to where we installed the custom Lua.
 
 ```sh
-$ luarocks install --local --only-deps pallene-dev-1.rockspec
+wget https://luarocks.org/releases/luarocks-3.9.0.tar.gz
+tar xf luarocks-3.9.0.tar.gz
+cd luarocks-3.9.0
+./configure --with-lua=/usr/local
+make
+sudo make install
 ```
 
-If you use the --local flag when installing packages from Luarocks, you may
-also need to configure the appropriate environment variables on your terminal configuration file.
-If you are using bash you can do (as stated in `luarocks --help path`):
+By default, Luarocks installs packages to the root directory, which requires sudo.
+If you are like me, you might prefer to intall to your home directory by default.
+
 ```sh
-$ echo 'eval `luarocks path`' >> ~/.bashrc
+# Run this one time
+luarocks config local_by_default true
 ```
-For further information, consult the [Luarocks documentation](https://github.com/luarocks/luarocks/wiki/path).
 
-### Compiling the custom interpreter
+Remember that in order for the local rocks tree to work, you must to set some environment variables
 
-Pallene must be run against a custom-built version of the Lua interpreter.
-This custom version of Lua 5.4 doesn't have to be the same one that you will use to run the compiler itself,
-or to install the Luarocks packages.
+```sh
+# Put this in your .bashrc
+eval "$(luarocks path)"
```

Comment:
(Alternatively)
```sh
echo "eval\"$(luarocks path)\"" >> ~/.bashrc
```

Reply:
I think I prefer telling people to do this by hand, because with the `>>` it is easy to run it more than once and end up with multiple redundant lines inside the bashrc. In any case, I tweaked the wording to make it just a little bit clearer.

Reply:
Gotcha